### PR TITLE
Add `isDirty` functionality

### LINKED ISSUE #3: Add `isDirty` command
Removed this part from #2 , and added a proper issue to it :) 

Maybe we should add something like this?:

```elm
isDirtyWith : Editable record -> (record -> record -> Bool) -> Bool
isDirtyWith editableRecord compareFunc =
    case editableRecord of
        ReadOnly _ ->
            False

        Editable originalRecord updatedRecord ->
            compareFunc originalRecord originalRecord


isDirty : Editable record -> Bool
isDirty editableRecord =
    isDirtyWith editableRecord (==)
```

### PR DESCRIPTION
**Pull request by amitaibu**
fixes #3 

Still WIP, but opened for possible early review.

@stoeffel seems there's no Travis integration, and I wasn't able to execute tests with `elm-test ./Tests.elm` -- could you indicate how you run the tests?

(Travis integration could be added in a follow up probably)

### CODE REVIEW

**Comment by stoeffel** [comment]
I just added elm-verify-examples. Sorry, for the merge conflicts.
You can run the tests like this:
`elm-verify-examples && elm-test`

**Comment by amitaibu** [comment]
`elm-verify-examples` is lovely!

editable_ _amitai_itamars-imac_ _-zsh_ _102x33


Ready for review

File: src/Editable.elm [hunk 1]
```diff
@@ -7,12 +7,14 @@ module Editable
         , save
         , update
         , value
+        , isDirty
```
Comment (amitaibu) [inline src/Editable.elm]: oh actually, I'll need to order by alpha-bet

Comment (stoeffel) [inline src/Editable.elm]: I think the latest elm-format should order these correctly

Comment (stoeffel) [inline src/Editable.elm]: I will fix this.

Comment (amitaibu) [inline src/Editable.elm]: Thanks!

**Comment by stoeffel** [comment]
published with 1.2.0 http://package.elm-lang.org/packages/stoeffel/editable/1.2.0/Editable

**Comment by stoeffel** [comment]
Thanks for your contribution!

**Pull request merged**